bin/select_secondary

```diff
@@ -27,7 +27,10 @@ ap.add_argument("-s", "--separation", type=float, default=1.,
 ap.add_argument("--scnddir",
                 help="Base directory of secondary target files (e.g. '/project/projectdirs/desi/target/secondary' at NERSC). "+
                 "Defaults to SCND_DIR environment variable.")
-
+ap.add_argument("--darkbright",
+                action='store_true',
```

Comment:
I think we want the default behaviour, here, to be to run with `darkbright=True`. In other words, we want the standard behavior to be to split everything into `bright-` and `dark-` file types. Can you change this so the argument is `--writeall` (like for the primary targets), and have the default be to split?

If you think `--writeall` is misleading, feel free to come up with a better term (but also change to that term in `select_targets` and `select_sv_targets` for the primary targets).

Reply:
OK, I was unsure what the default should be. Will fix this as suggested.